clarify signature-agent as an url (#85)
thibmeu/http-message-signatures-directory: draft-meunier-web-bot-auth-architecture.md

This section describes the discovery mechanism for the agent directory.

> The reference for discovery is an HTTPS URL whose host component is a FQDN. It SHOULD provide a directory hosted on the well known registered in Section 4 of {{DIRECTORY}}.

Reviewer: added some slight tuning. mostly, that we indeed use a URL and not an FQDN. That is SHOULD use HTTPS (the production angle)

```suggestion
The reference for discovery is a URL. It SHOULD be an HTTPS URL. It SHOULD provide a directory hosted on the well known registered in Section 4 of {{DIRECTORY}}.
```
